data/test/markdown_single_sections/incorrect_nlu_format.md

```diff
@@ -0,0 +1,4 @@
+## deny
```

Comment:
optional / up to you: Inline this to make the test more concise (it avoid having to jump to the fixture and then the file and then back to the test) and avoid us having a ton of files in the `data/test/` directories.

Reply:
I thought about that in the first place as well.
The thing is this test is testing the "train" command e2e, so it actually has to read a real file and open() it.